file_handling.py

```diff
@@ -0,0 +1,60 @@
+import os.path
+import json
+from constants import file_json_name
+from convert_file import convert_to_local
+
+def check_file_exists():
+    return os.path.isfile(file_json_name)
+
+def create_json_file():
+    with open(file_json_name, 'w') as f:
+        json.dump({}, f)
+
+def pokemon_by_id_in_json(pokemon_id, data):
+    try:
+        return data[str(pokemon_id)] 
```

Comment:
Do you return the content or the flag that states if the content is there? \
Also, why the `str()`?